sdk/core/Cargo.toml

```diff
@@ -27,16 +27,16 @@ reqwest = { version = "0.12.0", features = [
 ], default-features = false, optional = true }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-quick-xml = { version = "0.31", optional = true, features = ["serialize", "serde-types"] }
+quick-xml = { version = "0.31", features = ["serialize", "serde-types"] }
```

Comment:
`azure_core` is used by all crates, so it would be nice if `xml` was still an opt-in feature and you could condition XML support on the feature, which identity could default opt into if needed.

Still, if it's available in the headers, that not only seems more straight forward but faster at runtime.

Reply:
Thanks! I implemented it as suggested, didn't consider that the individual crates can require the optional features of the core.

I don't think adding the `xml` feature from the identity sdk is necessary as to my knowledge only the storage API returns XML responses, please correct me if I'm wrong.